maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -18,6 +18,12 @@
                 <include>**/*</include>
             </includes>
         </fileSet>
+        <fileSet filtered="true">
+            <directory></directory>
+            <includes>
+                <include>.env</include>
+            </includes>
+        </fileSet>
```

Comment:
This is the  trick to include hidden files in the archetype. 
There is an issue about .gitignore file as well. I will PR for the .gitignore separately.